src/subscription/subscription.service.ts

```diff
@@ -105,4 +129,23 @@ export class SubscriptionService {
 
         return deleteResult;
     }
+
+    async deleteBySubAndGrantId(
+        sub: string,
+        contentfulGrantSubscriptionId: string,
+    ): Promise<DeleteResult> {
```

Comment:
This type can be inferred
```suggestion
    ) {
```